src/Instrumentation/PDO/src/PDOTracker.php

```diff
@@ -106,4 +107,35 @@ private static function mapDriverNameToAttribute(?string $driverName): string
             default => 'other_sql',
         };
     }
+
+    /**
+     * Extracts attributes from a DSN string
+     *
+     * @param string $dsn
+     * @return array
+     */
+    private static function extractAttributesFromDSN(string $dsn): array
+    {
+        $attributes = [];
+        if (str_starts_with($dsn, 'sqlite')) {
+            $attributes[TraceAttributes::DB_NAME] = $dsn;
```

Comment:
Should the `sqlite:` prefix be dropped, assuming `TraceAttributes::DB_SYSTEM` would be `sqlite` already?

Reply:
I wanted to avoid an additional extraction dance for the database name - we can probably strip the prefix though